src/util/downloadCSV.js

```diff
@@ -8,8 +10,9 @@ function downloadCSV(url, stripes, params, mocks) {
   return fetch(`${cleanedURL}&csv=true`, {
     headers: {
       'X-Okapi-Tenant': stripes.okapi.tenant,
-      'X-Okapi-Token': stripes.okapi.token,
-    }
+      ...getLegacyTokenHeader(stripes.okapi),
+    },
+    credentials: 'include',
```

Comment:
```suggestion
      ...(stripes.okapi.token ? { 'X-Okapi-Token': stripes.okapi.token } : {}),
    },
    credentials: 'include',
```

Reply:
Thanks @alb3rtino !
Outsourcing of `getLegacyTokenHeader` is build back